libraries/AP_HAL/examples/RCOutput/RCOutput.cpp

```diff
@@ -6,9 +6,13 @@
 
 const AP_HAL::HAL& hal = AP_HAL::get_HAL();
 
+#define RC_SPEED 50
+
 void setup (void) 
 {
     hal.console->println("Starting AP_HAL::RCOutput test");
+    hal.rcout->set_freq(0xFF, RC_SPEED);   // Set RC Speed
```

Comment:
I think this should be optional. We use this example to test PCA9685 and other new RCOutput implementations while doing bench tests.

Probably integrating the AP_Menu to set the frequency would be the best idea. See https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_Linux/examples/GPIOTest/GPIOTest.cpp for a reference implementation using it.